src/plugins/login/loginPlugin.test.ts

```diff
@@ -56,6 +57,7 @@ describe("Login Plugin", () => {
     AzureLoginService.servicePrincipalLogin = servicePrincipalLogin;
     
     const sls = MockFactory.createTestServerless();
+    delete sls.variables['azureCredentials']
```

Comment:
Can we move these into a `beforeEach` or similar to keep it DRY?

Reply:
The only issue with that is the deletion of the credentials is specific to these tests only. Other tests depend on the credentials still being there